single-asset-staking.cabal

```diff
@@ -171,7 +173,11 @@ test-suite single-asset-staking-test
   build-depends:  single-asset-staking
   main-is:        Spec.hs
   hs-source-dirs: test
-  other-modules:  Spec.StakingSpec
+  other-modules:
+    Conversions
+    Spec.StakingSpec
+
+  hs-source-dirs: src-optimized
```

Comment:
Should we also add the `optimized` flag and these `hs-source-dirs` conditionally?

Reply:
We would want to be using the src-optimized flag cause I'm facing this error (ptryFrom(PPubKeyHash): must be 28 bytes long") when using pconvert from src-checked for StakingSpec. I'm passing a BuiltInByteString which is 56 characters in hex i.e. 28 bytes still..